content/applications/sales/sales/products_prices/prices/pricing.rst

```diff
@@ -317,6 +317,43 @@ product form.
 
 .. _sales/advanced-price-rules:
 
+Discounts
+=========
+
+The :guilabel:`Discounts` feature allows the ability to set a discount or increase the price
+on *individual items* in a sales order. This is calculated as a percentage of that product's sales
+price.
```

Comment:
Nitpick– :guilabel:s should only be placed on text that is present on the current UI– and since the navigation instruction comes later, we can use italics to refer to the feature :)
```suggestion
The *Discounts* feature allows the ability to set a discount or increase the price on *individual
items* in a sales order. This is calculated as a percentage of that product's sales price.
```